src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -46,14 +47,24 @@ class MovingWindow:
     [`OrderedRingBuffer`][frequenz.sdk.timeseries._ringbuffer.OrderedRingBuffer]
     documentation.
 
+    Resampling might be required when working with streams of samples from
+    different sources at different sampling rates or periods, and it can be
```

Comment:
This is not quite accurate. At this point in time streams are already combined. Here it's mostly useful to reduce the amount of data.

Reply:
You're right, I'll amend it and probably need to amend the release notes

Reply:
Amended